### PR TITLE
StringSimilarity None Exception

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-27085

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_12_14.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### StringSimilarity
+
+- %%UPDATE_RN%%
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_12_14.md:6]:
```suggestion

```

File: Packs/CommonScripts/ReleaseNotes/1_12_14.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Updated the Docker image to: *demisto/python3:3.10.12.68714*.
+- Fixed an issue where an exception could be raised when string are similar.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_12_14.md:8]:
```suggestion
- Fixed an issue where an exception could be raised when strings are similar.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

File: Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py [hunk 3]
```diff
@@ -20,10 +20,13 @@
         "StringB": "test",
         "SimilarityScore": 0.7272727272727273,
     }),
+    ("foo", "bar", 0.1, {}),
 ])
 def test_string_similarity(first_string, second_string, similarity_threshold, expected_result, mocker):
     try:
         result = stringSimilarity(first_string, second_string, similarity_threshold)
+        print(f'{result.outputs=}')
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py:28]: Why print the results?

**Review by DeanArbel — APPROVED** [review]

File: Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py (hunk 3 above)
Comment (tkatzir) [reply Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity_test.py:28]: Left there by mistake. Used for debugging. Removed.

**Pull request closed without merging**